### PR TITLE
8284278: Diagnostic Fix Updated.

### PR DESCRIPTION
**Pull request by TejeshR13**
Diagnostic Fix added for the Bug - [JDK-8283070](https://bugs.openjdk.java.net/browse/JDK-8283070)
javax/swing/JTable/8236907/LastVisibleRow.java failed with "RuntimeException: Test Case Failed!!". 
The bug was not reproducible in local machine and with runs on mach5. It was however observed in mach5 testing intermittently. The LastVisibleRow.java captures two Images before and After Mouse click of last Row in a Table. The Test Fails if the two images does not match. In order to find the root cause, the two images are written to a file for diagnostic purpose.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8284278](https://bugs.openjdk.java.net/browse/JDK-8284278): Diagnostic Fix Updated.

### Reviewers
 * [Phil Race](https://openjdk.java.net/census#prr) (@prrace - **Reviewer**) ⚠️ Review applies to 063ccc00f36807b4cfb24fa82b44f10397c81737
 * [Prasanta Sadhukhan](https://openjdk.java.net/census#psadhukhan) (@prsadhuk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8088/head:pull/8088` \
`$ git checkout pull/8088`

Update a local copy of the PR: \
`$ git checkout pull/8088` \
`$ git pull https://git.openjdk.java.net/jdk pull/8088/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8088`

View PR using the GUI difftool: \
`$ git pr show -t 8088`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8088.diff">https://git.openjdk.java.net/jdk/pull/8088.diff</a>

</details>

### CODE REVIEW

**Review by prrace — APPROVED** [review]

File: test/jdk/javax/swing/JTable/8236907/LastVisibleRow.java [hunk 1]
```diff
@@ -75,9 +77,12 @@ public void run() {
             clearSelect();
             testRobot.waitForIdle();
             BufferedImage bufferedImageAfter = testRobot.createScreenCapture(getCaptureRect());
+
             testRobot.delay(1000);
 
             if (!compare(bufferedImageBefore, bufferedImageAfter)) {
+                ImageIO.write(bufferedImageBefore, "png", new File("bufferedImageBefore.png"));
+                ImageIO.write(bufferedImageAfter, "png", new File("bufferedImageAfter.png"));
                 throw new RuntimeException("Test Case Failed!!");
```
Comment (prsadhuk) [inline test/jdk/javax/swing/JTable/8236907/LastVisibleRow.java:86]: Probably you can change the failure message also now which I guess should be something like "Last row is not visible" 
Also, I see from the screen.png artifacts generated when the test failed shows a empty desktop with "Home" "Trash" icons on lower right of screen so not sure if this generated images also will help but let's try..

Comment (TejeshR13) [reply test/jdk/javax/swing/JTable/8236907/LastVisibleRow.java]: Done.....

**Review by prsadhuk — APPROVED** [review]

**Pull request closed without merging**